code/main/media/MediaSession.brs

```diff
@@ -210,26 +221,15 @@
         end function,
 
         _attachMediaConfig: function(xdmData as object) as object
-            xdmData.xdm["mediaCollection"]["sessionDetails"]["playerName"] = m._configurationModule.getMediaPlayerName()
-            xdmData.xdm["mediaCollection"]["sessionDetails"]["channel"] = m._configurationModule.getMediaChannel()
+            xdmData.xdm["mediaCollection"]["sessionDetails"]["playerName"] = m._getPlayerName()
+            xdmData.xdm["mediaCollection"]["sessionDetails"]["channel"] = m._getChannelName()
```

Comment:
We can add the sessionConfig check directly here. Instead of adding helpers for all.